packages/chrome/src/elements/subnav/CollapsibleSubNavItem.tsx

```diff
@@ -18,8 +18,15 @@ import {
 
 export interface ICollapsibleSubNavItemProps
   extends Omit<ButtonHTMLAttributes<HTMLButtonElement>, 'onChange'> {
+  /** Sets the header */
   header?: React.ReactNode;
+  /** Expands the nav item */
   isExpanded?: boolean;
```

Comment:
This prop does not perform the act of expanding the nav item. The `onChange` prop is responsible for that. Rather, it indicates whether the nav item is expanded.

Reply:
Updated in latest push.